Dockerfile

```diff
@@ -80,6 +84,7 @@ FROM base AS dynamic-link
 
 COPY --from=dynamic-build /opt /opt
 
+LABEL org.opencontainers.image.build_target="dynamic-link"
```

Comment:
Not a standard [Open Containers label](https://specs.opencontainers.org/image-spec/annotations/), right?

Reply:
Yep, missed this part when editing the commit, tnks!